### PR TITLE
Fix bug with readme to named file generation

### PR DESCRIPTION
**Pull request by mrbobbytables**
This fixes a bug with relocating files, it was originally a WIP to demo syncing the sig list but that will be added in a later PR once things are updated in k/community.

### CODE REVIEW

File: external-sources/kubernetes/community [hunk 1]
```diff
@@ -1 +1,37 @@
...
+"/sig-ui/README.md","/groups/sig-ui.md"
+"/sig-usability/README.md","/groups/sig-usability.md"
+"/sig-windows/README.md","/groups/sig-windows.md"
+"/ug-big-data/README.md","/groups/ug-big-data.md"
+"/ug-vmware-users/README.md","/groups/ug-vmware-users.md"
+"/wg-apply/README.md,"/groups/wg-apply.md"
+"/wg-component-standard/README.md","/groups/wg-component-standard.md"
+"/wg-data-protection/README.md","/groups/wg-data-protection.md"
+"/wg-iot-edge/README.md","/groups/wg-iot-edge.md"
+/wg-k8s-infra/README.md","/groups/wg-k8s-infra.md"
```
Comment (justaugustus) [inline external-sources/kubernetes/community]:
```suggestion
"/wg-k8s-infra/README.md","/groups/wg-k8s-infra.md"
```

Comment (mrbobbytables) [reply external-sources/kubernetes/community]: Thanks! 
fixed 👍

File: external-sources/kubernetes/community [hunk 2]
```diff
@@ -1 +1,37 @@
...
+"/wg-apply/README.md,"/groups/wg-apply.md"
+"/wg-component-standard/README.md","/groups/wg-component-standard.md"
+"/wg-data-protection/README.md","/groups/wg-data-protection.md"
+"/wg-iot-edge/README.md","/groups/wg-iot-edge.md"
+"/wg-k8s-infra/README.md","/groups/wg-k8s-infra.md"
+"/wg-lts/README.md","/groups/wg-lts.md"
+"/wg-machine-learning/README.md","/groups/wg-machine-learning.md"
+"/wg-policy/README.md","/groups/wg-policy.md"
+"/wg-resource-management/README.md","/groups/wg-resource-management.md"
+"/wg-security-audit/README.md","/groups/wg-security-audit.md"
```
Comment (vishakhanihore) [inline external-sources/kubernetes/community]: What about having `Committee-List` contents too?

File: config.toml [hunk 3]
```diff
@@ -22,7 +22,7 @@ pygmentsStyle = "dracula"
       autoHeadingIDType = "github"
     [markup.goldmark.renderer]
       hardWraps = false
-      unsafe = false
+      unsafe = true
```
Comment (vishakhanihore) [inline config.toml]: Is this needed? We don't have much inline js or Html in our contents.

Comment (mrbobbytables) [reply config.toml]: Yes, `sig-list.md` uses it and it'll be our one way to make certain parts of the site look a bit better. =/

**Comment by mrbobbytables** [comment]
I've removed the wip and made this PR just the bug fix to get it merged. Will do a PR later to add community groups after we fix ordering / naming in k/community

**Pull request merged**